lib/helpers/buildURL.js

```diff
@@ -2,7 +2,7 @@
 
 var utils = require('./../utils');
 
-function encode(val) {
+function encodeKey(val) {
   return encodeURIComponent(val).
```

Comment:
Why does this function exist at all? Isn't this just a poorly written home-rolled version of this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent

Which has been available since forever?

Even with the patch this will fail to properly encode | into %7C which is breaking apache for me as Auth0 uses | in it's userIds which we also use as surrogate primary keys in our database.